synapse/lib/stormtypes.py

```diff
@@ -780,6 +793,11 @@ async def _libPkgDeps(self, pkgdef):
         pkgdef = await toprim(pkgdef)
         return await self.runt.snap.core.verifyStormPkgDeps(pkgdef)
 
+    async def _libPkgVars(self, name):
```

Comment:
Should we only allow package vars for registered packages? It would make package vars for uninstalled packages inaccessible from storm but that might be ok?

Reply:
I think we could consider that once we figure out the cleanup steps, otherwise we could box ourselves in with not being able to remove old data for uninstalled pkgs if needed.